File: src/containers/Semana2/Desafio#4.js
function poligono(pol,val1,val2)
{
    if (pol=="Triangulo")
    {
        resultado=(val1*val2)/2
        console.log("El area del Triangulo es: ",resultado)
    }
    else if (pol=="Cuadrado")
    {
        resultado=val1*val2
        console.log("El area del Cuadrado es: ",resultado)

    }
    else if (pol=="Rectangulo")
    {
        resultado=val1*val2
        console.log("El area del Rectangulo es:",resultado)

    }
}
poligono("Rectangulo",4,3);

File: src/containers/Semana2/Desafio#4.py
def calcular(pol,valor1,valor2):
    if pol=="Triangulo":
        resultado=valor1*valor2/2
        print("El area del Triangulo es: ",resultado)
    elif pol=="Cuadrado":
        resultado=valor1*valor2
        print("El area del Cuadrado es: ",resultado)

    elif pol=="Rectangulo":
        resultado=valor1*valor2
        print("El area del Rectangulo es:",resultado)

calcular("Rectangulo",4,3)